lib/datadog/tracing/contrib/http/circuit_breaker.rb

```diff
@@ -29,6 +29,16 @@ def internal_request?(request)
           end
 
           def should_skip_distributed_tracing?(client_config)
+            if Datadog.configuration.appsec.standalone.enabled
+              active_trace = Tracing.active_trace
+
+              # return true if absence of upstream ASM event (_dd.p.appsec:1)
+              # and no local security event (which sets _dd.p.appsec:1 locally)
```

Comment:
again, I think it makes sense to explain here in which case `_dd.p.appsec` is being set to `1` and why we want to skip distributed tracing in such case

Reply:
I also added more context to this one